client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -135,27 +153,35 @@ const DataFilesCopyModal = React.memo(() => {
             <div className="dataFilesModalColHeader">
               Destination
               <DataFilesSystemSelector
-                systemId={modalParams.system}
+                operation="copy"
+                systemId={params.system}
                 section="modal"
                 disabled={disabled}
+                showProjects={showProjects}
               />
             </div>
-            <DataFilesBreadcrumbs
-              api={modalParams.api}
-              scheme={modalParams.scheme}
-              system={modalParams.system}
-              path={modalParams.path || '/'}
-              section="modal"
-            />
-            <div className="filesListing">
-              <DataFilesModalListingTable
-                data={files.filter(listingFilter)}
-                operationName="Copy"
-                operationCallback={copyCallback}
-                operationOnlyForFolders
-                operationAllowedOnRootFolder
-                disabled={disabled}
+            {!showProjects && (
+              <DataFilesBreadcrumbs
+                api={modalParams.api}
+                scheme={modalParams.scheme}
+                system={modalParams.system}
+                path={modalParams.path || '/'}
+                section="modal"
               />
+            )}
+            <div className="filesListing">
+              {showProjects ? (
+                <DataFilesProjectsList modal="copy" />
```

Comment:
![Screen Shot 2020-12-16 at 2 24 25 PM](https://user-images.githubusercontent.com/8287580/102406362-12f87980-3fb0-11eb-83af-e268e031ec97.png)

Some styling issues:
* text size
* scrolling when there is lots of workspaces
* long titles overflow
* Spacing between "Workspace Title" and "Destination"

Reply:
Adding these notes to the main PR